src/meta/src/model_v2/database.rs

```diff
@@ -14,23 +14,19 @@
 
 use sea_orm::entity::prelude::*;
 
+use crate::model_v2::DatabaseId;
+
 #[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
 #[sea_orm(table_name = "database")]
 pub struct Model {
     #[sea_orm(primary_key, auto_increment = false)]
-    pub database_id: i32,
+    pub database_id: DatabaseId,
```

Comment:
Is this also generated by `sea-orm-cli`? Or it's manually tweaked?

Reply:
It's manually tweaked to keep consistency of our previous data type, which can avoid many explicit casting.

Reply:
Got it. So developers in the future may need to take care of this instead of copying the generated files mindlessly. 🥵

Reply:
The file generated based on the database is required only during the initial development of sqlbackend. Subsequent changes are very low frequency. According to my recent development, I only need to change the corresponding column changes and so on, which is not very error-prone. In addition except to adding new tables, it is recommended to modify these files manually.